src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -676,8 +679,35 @@ long IFloatingPoint<double>.GetExponentShortestBitLength()
             }
         }
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.GetExponentByteCount()" />
-        int IFloatingPoint<double>.GetExponentByteCount() => sizeof(short);
+        /// <inheritdoc cref="IFloatingPoint{TSelf}.GetSignificandByteCount()" />
+        int IFloatingPoint<double>.GetSignificandByteCount() => sizeof(ulong);
```

Comment:
Isn't the answer actually 7?  `(GetSignificandBitLength() + 7) / 8` => 7.

I guess as an implementation detail we just don't support omitting the extraneous byte?

Reply:
Right, this gets into what I was just covering in https://github.com/dotnet/runtime/pull/69391#discussion_r874052826

It is undesirable to return the shortest sequence of bytes because it makes it immensely more complex to work with the underlying data. Returning `8-bytes` here allows devs to simply get the significand out as a `long` or `ulong` and then operate on it however they need.